src/libcpu/src/jit/jit.cpp

```diff
@@ -265,16 +379,25 @@ gen(JitBlock &block)
          a.bind(targetIter->second.label);
       }
 
-      if (JIT_DEBUG) {
-         a.mov(a.niaMem, lclCia + 4);
-      }
-
       auto instr = mem::read<espresso::Instruction>(lclCia);
       auto data = espresso::decodeInstruction(instr);
       auto genSuccess = false;
 
       a.genCia = lclCia;
 
+      // Don't attempt to verify non-repeatable instructions
+      bool canVerify = (data->id != espresso::InstructionID::kc
```

Comment:
Probably should move this inside the if statement below to avoid the cost of calculating it when the verifier is off.